src/main/java/org/betonquest/betonquest/menu/MenuItem.java

```diff
@@ -83,7 +92,8 @@ public class MenuItem extends SimpleYMLSection {
 
     @SuppressWarnings({"PMD.ExceptionAsFlowControl", "PMD.CyclomaticComplexity", "PMD.CognitiveComplexity", "PMD.NPathComplexity"})
     @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
-    public MenuItem(final QuestPackage pack, final String name, final ConfigurationSection section) throws InvalidConfigurationException {
+    public MenuItem(final QuestPackage pack, final String menuId, final String name, final ConfigurationSection section)
```

Comment:
Does I see it right, that the menuId was added for better exxeption messages?

Reply:
Is that ok? I asked for it for testing and told him to leave it? Maybe it needs worded better for consistency?

Reply:
Please then pass the MenuID object instead of only the String ;)